MY_CPU_BE expands to an empty token sequence, so defined (MY_CPU_BE)
has to be used in this context.  This code is only reachable for
s390x, ppc64 sets MY_CPU_NAME earlier.

diff -ur evince-3.25.4.orig/cut-n-paste/unarr/lzmasdk/CpuArch.h evince-3.25.4/cut-n-paste/unarr/lzmasdk/CpuArch.h
--- evince-3.25.4.orig/cut-n-paste/unarr/lzmasdk/CpuArch.h	2017-07-22 07:58:23.000000000 +0200
+++ evince-3.25.4/cut-n-paste/unarr/lzmasdk/CpuArch.h	2017-07-31 13:32:33.893245858 +0200
@@ -174,7 +174,7 @@
 #ifndef MY_CPU_NAME
   #ifdef MY_CPU_LE
     #define MY_CPU_NAME "LE"
-  #elif MY_CPU_BE
+  #elif defined (MY_CPU_BE)
     #define MY_CPU_NAME "BE"
   #else
     /*
Only in evince-3.25.4/cut-n-paste/unarr/lzmasdk: CpuArch.h~
